core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-selected/dot-category-field-selected.component.html

```diff
@@ -0,0 +1,32 @@
+<ul class="category-list">
+    @for (category of $categories(); track category.key) {
+    <li class="category-list__item" @fadeAnimation data-testId="category-item">
+        <div class="category-list__item-content">
+            <div class="category-list__title" data-testId="category-title">
+                {{ category.value }}
+            </div>
+            <div class="category-list__path">
+                <span
+                    [pTooltip]="category.path"
+                    tooltipPosition="top"
+                    class="category-list__path-content"
+                    data-testId="category-path">
+                    {{ category.path || 'edit.content.category-field.category.root-name' | dm }}
+                </span>
+            </div>
+        </div>
+
+        <div class="category-list__remove">
+            <button
+                (click)="removeItem.emit(category.key)"
+                icon="pi pi-times"
+                class="p-button-sm p-button-text p-button-secondary"
+                data-testId="category-remove-btn"
+                pButton></button>
+        </div>
+    </li>
+
+    } @empty {
+    <li data-testId="category-list-empty">No Categories selected</li>
```

Comment:
Does this need translation?

Reply:
No, we have a issue for implement the final component of empty here.